conans/cli/commands/help.py

```diff
@@ -32,25 +32,26 @@ def output_help_cli(out, commands, groups):
                 data.append(line)
 
             txt = textwrap.fill(' '.join(data), 80, subsequent_indent=" " * (max_len + 2))
-            out.writeln(txt)
+            out.info(txt)
 
-    out.writeln("")
-    out.writeln('Conan commands. Type "conan <command> -h" for help', Color.BRIGHT_YELLOW)
+    out.info("")
+    out.info('Conan commands. Type "conan <command> -h" for help', Color.BRIGHT_YELLOW)
 
 
 @conan_command(group="Misc", formatters={"cli": output_help_cli})
-def help(*args, conan_api, parser, commands, groups, **kwargs):
+def help(conan_api, parser, *args, **kwargs):
```

Comment:
Why this change? I prefer to have the parameters listed than to extract them from the `**kwargs` parameter